File: 2577 숫자의 개수/2577.js
let fs = require('fs');
let inputs = fs.readFileSync('/dev/stdin').toString().split('\n');
inputs.pop()

let number = 1;
inputs.forEach(input => number *= parseInt(input));

let numberStr = String(number);
let numberStrLen = numberStr.length;
let answers = [];
for(let i=0;i<10;++i){
    answers.push(parseInt(0))
}
for (let i = 0; i < numberStrLen; ++i) {
    ++answers[numberStr[i].charCodeAt(0) - '0'.charCodeAt(0)];
}

for (let i = 0; i < 10; ++i) {
    console.log(answers[i]);
}

File: 2577 숫자의 개수/2577.py
number = 1
for i in range(0, 3):
    number *= int(input())

numberStr = str(number)
answers = [0 for _ in range(0, 10)]
for character in numberStr:
    answers[ord(character) - ord('0')] += 1

for answer in answers:
    print(answer)